mayastor/src/grpc/nexus_grpc.rs

```diff
@@ -73,6 +73,22 @@ impl Nexus {
             rebuilds: RebuildJob::count() as u32,
         }
     }
+
+    pub fn to_grpc_v2(&self) -> rpc::NexusV2 {
```

Comment:
Should this be an impl From<&Nexus> to rpc::NexusV2 ?

Reply:
There's actually a code comment above `to_grpc(&self)`:
```
    /// We cannot use From trait because it is not value to value conversion.
    /// All we have is a reference to nexus.
```

Reply:
hmm that comment doesn't seem quite right, you can have traits for references?

Reply:
Seems to the Jan's comment. Anyway, if I do have an `impl From<&Nexus> for rpc::NexusV2` and try to use an `into`:
```
error[E0277]: the trait bound `NexusV2: From<&Box<nexus_bdev::Nexus>>` is not satisfied
   --> mayastor/src/grpc/mayastor_grpc.rs:652:32
    |
652 |                     .map(|n| n.into())
    |                                ^^^^ the trait `From<&Box<nexus_bdev::Nexus>>` is not implemented for `NexusV2`
    |
    = help: the following implementations were found:
              <NexusV2 as From<&nexus_bdev::Nexus>>
    = note: required because of the requirements on the impl of `Into<NexusV2>` for `&Box<nexus_bdev::Nexus>`
```
You can try double-dereferencing the `&Box` to get a `nexus_bdev::Nexus`, then you either change to `impl From<&Nexus>` and get a missing Copy trait, or try `&(**n).into()` and get `expected struct ``NexusV2``, found reference`. I'm just going to leave it alone.

Reply:
Box impl's deref, so just need to ".map(|n| n.deref().into()".
But yeah leave it at that for now